packages/eslint-plugin/docs/rules/no-misused-promises.md

```diff
@@ -45,15 +67,17 @@ expected to be checked, your configuration will look like this:
 }
 ```
 
-Likewise, if you don't want to check conditionals, you can configure the rule
-like this:
+You can disable selective parts of the `checksVoidReturn` option by providing an
+object that disables specific checks:
```

Comment:
we should probably add a quick description / example for each option.
it's verbose but will help people self-service